privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -127,7 +127,31 @@ def submit_message(self, firebase_token, data):
         fcm_message = {
             "message": {
                         "data": data,
-                        "token": firebase_token
+                        "token": firebase_token,
+                        "android": {
+                                    "priority": "HIGH",
```

Comment:
The android part adds nothing, it could be omitted.

Reply:
In my testing it will actually stop the notification from hanging around in the push network, if the phone is offline. That's a small improvement, but I think we should keep it nevertheless.